packages/main/src/TableSelection.ts

```diff
@@ -77,6 +77,7 @@ class TableSelection extends UI5Element implements ITableFeature {
 	@property()
 	selected = "";
 
+	readonly _identifier = "TableSelection";
```

Comment:
This only works with TableSelection specifically right? If you'd create a Selection feature yourself by extending the TableSelection, the tag would be different and this check would fail again

Reply:
not really if you add the attribute in onDomEnter then the one that inherits gets the attribute as well.

I see this change as a part of the table feature concept for importing features separately therefore those semi-privates are not welcome. As mentioned above we can add the identifier/discriminator to the ITableFeature interface then I am fine.

Reply:
Done